From 047d323452401a633118bbff3eae45dcb49096ba Mon Sep 17 00:00:00 2001
From: Mauro Carvalho Chehab <mchehab@s-opensource.com>
Date: Thu, 22 Feb 2018 12:54:28 -0500
Subject: [PATCH 035/437] tda1997x: get rid of an unused var

1 warning regressions:
  + drivers/media/i2c/tda1997x.c: warning: variable 'last_irq_status' set but not used [-Wunused-but-set-variable]:  => 1421:17

Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>
---
 drivers/media/i2c/tda1997x.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/i2c/tda1997x.c b/drivers/media/i2c/tda1997x.c
index a480aafecbf6..3021913c28fa 100644
--- a/drivers/media/i2c/tda1997x.c
+++ b/drivers/media/i2c/tda1997x.c
@@ -1418,13 +1418,13 @@ static void tda1997x_irq_rate(struct tda1997x_state *state, u8 *flags)
 	struct v4l2_subdev *sd = &state->sd;
 	u8 reg, source;
 
-	u8 irq_status, last_irq_status;
+	u8 irq_status;
 
 	source = io_read(sd, REG_INT_FLG_CLR_RATE);
 	io_write(sd, REG_INT_FLG_CLR_RATE, source);
 
 	/* read status regs */
-	last_irq_status = irq_status = tda1997x_read_activity_status_regs(sd);
+	irq_status = tda1997x_read_activity_status_regs(sd);
 
 	/*
 	 * read clock status reg until INT_FLG_CLR_RATE is still 0
-- 
2.16.3

